src/snowflake/snowpark/functions.py

```diff
@@ -2886,6 +3019,26 @@ def cume_dist() -> Column:
     """
     Finds the cumulative distribution of a value with regard to other values
     within the same window partition.
+
+    Example:
+
+        >>> from snowflake.snowpark.window import Window
+        >>> df = session.create_dataframe([[1, 2], [1, 2], [1,3], [4, 5], [2, 3], [3, 4], [4, 7], [3,7], [4,5]], schema=["a", "b"])
+        >>> df.select(cume_dist().over(Window.order_by("a")).alias("result")).show()
+        ----------------------
+        |"RESULT"            |
+        ----------------------
+        |0.3333333333333333  |
+        |0.3333333333333333  |
+        |0.3333333333333333  |
+        |0.4444444444444444  |
+        |0.6666666666666666  |
```

Comment:
I'm surprised that this does not end with `7` lol. I don't have a straightforward way to make this result not use long float number :/